src/Sylius/Bundle/ApiBundle/CommandHandler/PickupCartHandler.php

```diff
@@ -131,7 +131,10 @@ private function getLocaleCode(?string $localeCode, ChannelInterface $channel):
         }
 
         if (!$this->hasLocaleWithLocaleCode($channel, $localeCode)) {
-            throw new \InvalidArgumentException('Choosen localeCode doesn\'t exist.');
+            throw new \InvalidArgumentException(sprintf(
+                'Cannot pick up cart, locale code "%s" does not exist.',
```

Comment:
`Choosen` should have been `Chosen`, but I changed the message to be a bit more explict.